bigbluebutton-web/deploy_to_usr_share.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+cd "$(dirname "$0")"
+sudo service bbb-web stop
+./build.sh
+
+grails assemble
+mkdir exploded && cd exploded
+jar -xvf ../build/libs/bigbluebutton-0.10.0.war
+cp ../run-prod.sh .
+sudo cp -R /usr/share/bbb-web /usr/share/bbb-web-old
```

Comment:
I only copy to bbb-web-old once, the first time I start working on bbb-web. Otherwise (repetititve re-deploy of .war) would cause me losing the original /usr/share/bbb-web